tests/queries/0_stateless/01699_timezoneOffset.sql

```diff
@@ -0,0 +1,20 @@
+
+/* timestamp  ==  (Europe/Moscow) */
+
+
+SELECT number,(toDateTime('1981-04-01 00:00:00', 'Europe/Moscow') + INTERVAL number * 600 SECOND) AS k, timezoneOffset(k) AS t, toUnixTimestamp(k) as s FROM numbers(200);
```

Comment:
It would be good if test result is readable. Now, when it has > 1k rows it is not. Can we just use, e.g., 2 rows instead of 200?

Reply:
The most interesting cases are the cases when daylight saving time have changed. With time just before and just after change. Also compare result with year with permanent DST (e.g. from 2011 to 2014 in Russia). Please, add comment if needed.

Reply:
have made modifications based on your opinion
thanx~